ontology/EBUCore/ebucore.owl

```diff
@@ -2719,14 +2708,6 @@ ec:offers rdf:type owl:ObjectProperty ;
           rdfs:label "PublicationEvent"@en .
 
 
-###  http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#partId
```

Comment:
Why it is removed?

Reply:
The concept is covered by the identificator instance attributed to the part.